src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Invoices/Blocks/generate_by_status.html.twig

```diff
@@ -64,11 +57,12 @@
       {% set label_attr = label_attr|merge({class: (label_attr.class|default('') ~ ' ' ~ parent_label_class)|trim}) %}
     {% endif %}
     <div class="md-checkbox">
-      <label{% for attrname, attrvalue in label_attr %} {{ attrname }}="{{ attrvalue }}"{% endfor %}>
+      <label>
         {{- widget|raw -}}
         <i class="md-checkbox-control"></i>
         {% set badgeType = orders_count > 0 ? 'success' : 'danger' %}
         <span class="badge badge-{{ badgeType }}">{{ orders_count }}</span>
+        &nbsp;
         <span class="status-name">
```

Comment:
After simplification whitespace no longer appeared between badge and status-name.

Reply:
👍